### DISCUSSION TITLE
Invaild header field

### DISCUSSION THREAD

**Discussion by lihaonet** [pwh-pwh/aiwechat-vercel#17] [General]
Post "https://api.openai.com/v1/chat/completions/chat/completions": net/http: invalid header field value for "Authorization"

**Comment by pwh-pwh** [comment]
这个错误怎么来的

**Reply by lihaonet** [reply]
Vercel部署完，我访问/api/chat 那个接口就显示这个问题

**Comment by pwh-pwh** [comment]
url或者是token配置问题

**Reply by lihaonet** [reply]
感谢，解决了是token问题，超过配额了
